pkgs/applications/editors/rstudio/update.sh

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env nix-shell
+#!nix-shell -i bash -p nix-update yarn2nix
+
+set -euo pipefail
+
+nixpkgsTop="$(git rev-parse --show-toplevel)"
+rstudioDir="$nixpkgsTop/pkgs/applications/editors/rstudio"
```

Comment:
I'd use this script's directory as the `rstudioDir`, with:

```
dirname="$(dirname "$0")"
```

It'll work for a nixpkgs from a tarball and not from a git checkout.